plugins/Vibed/VibratingString.cpp

```diff
@@ -99,7 +100,7 @@ void VibratingString::resample(const float* src, f_cnt_t srcFrames, f_cnt_t dstF
 	{
 		const float srcFrameFloat = frame * static_cast<float>(srcFrames) / dstFrames;
 		const float fracPos = srcFrameFloat - static_cast<f_cnt_t>(srcFrameFloat);
-		const f_cnt_t srcFrame = std::clamp(static_cast<f_cnt_t>(srcFrameFloat), 1, srcFrames - 3);
+		const f_cnt_t srcFrame = std::clamp<f_cnt_t>(static_cast<f_cnt_t>(srcFrameFloat), 1, srcFrames - 3);
```

Comment:
I'd rather avoid specifying the template argument, and instead make sure we're passing values of the right type. This ensures the template always uses the appropriate type for the arguments, and avoids implicit conversions (in fact, if you're manually specifying the template argument to fix an error, you're almost certainly relying on such conversions).
```suggestion
		const f_cnt_t srcFrame = std::clamp(static_cast<f_cnt_t>(srcFrameFloat), f_cnt_t{1}, srcFrames - 3);
```
There are a couple of other cases to address involving `clamp`, `max`, etc.